index.html

```diff
@@ -163,12 +171,36 @@ <h2>
       </dd>
       <dt>
         <dfn data-dfn-for="digital credential" data-local-lt=
-        "response">Presentation response</dfn>
+        "presentation response">Presentation response</dfn>
       </dt>
       <dd>
-        A format that a [=holder|holder's=] software, such as a digital wallet,
+        A format that a [=holder's=] software, such as a digital wallet,
         uses, via an [=digital credential/exchange protocol=], to respond to a
-        [=digital credential/request data=] by a [=verifier=].
+        [=digital credential/presentation request=] by a [=verifier=].
+      </dd>
+      <dt>
+        <dfn data-dfn-for="digital credential" data-local-lt="issuance">Issuance request</dfn>
+      </dt>
+      <dd>
+        An issuance request is a request to issue a [=digital credential=]
+        composed of an [=digital credential/credential offer=] and a [=digital
+        credential/exchange protocol=].
+      </dd>
+      <dt>
+        <dfn data-dfn-for="digital credential">credential offer</dfn>
+      </dt>
+      <dd>
+        A data structure that [=issuer=] or [=user agent=] software uses, via an
+        [=digital credential/exchange protocol=], to offer issuance of a [=digital
```

Comment:
I think we need a new definition for "issuance protocol" that can be referenced instead of [=digital credential/exchange protocol=].